tests/manage/mcg/test_namespace.py

```diff
@@ -30,6 +31,199 @@ class TestNamespace(MCGTest):
     # is resolved
     DEFAULT_REGION = "us-east-2"
 
+    @skipif_ocs_version("<4.7")
+    @tier1
+    @pytest.mark.parametrize(
+        argnames="nss_tup",
+        argvalues=[
+            pytest.param(("oc", {"aws": [(1, "eu-central-1")]})),
+            pytest.param(("oc", {"azure": [(1, None)]})),
+            pytest.param(("oc", {"rgw": [(1, None)]})),
+        ],
+        # A test ID list for describing the parametrized tests
+        # <CLOUD_PROVIDER>-<METHOD>-<AMOUNT-OF-BACKINGSTORES>
+        ids=[
+            "AWS-OC-1",
+            "AZURE-OC-1",
+            "RGW-OC-1",
+        ],
+    )
+    def test_namespace_store_creation_crd(self, namespace_store_factory, nss_tup):
+        """
+        Test namespace store creation using the MCG CRDs.
+        """
+        # Create the namespace store and verify health
```

Comment:
Health is not validated in this test (it is not validated in new factory function)

Reply:
Done